Description: increase the maximum possible value of the page size
 Firefox/Iceweasel uses a page size of 32k, but sqlite won't allow it
 because the maximum it allows by default is 8k (through the
 SQLITE_MAX_DEFAULT_PAGE_SIZE define). This patch increases this limit.
 .
Author: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
Bug-Debian: http://bugs.debian.org/672573

---
Forwarded: <not-needed>
Last-Update: <2012-05-16>

--- sqlite3-3.7.12.orig/src/sqliteLimit.h
+++ sqlite3-3.7.12/src/sqliteLimit.h
@@ -169,7 +169,7 @@
 ** SQLite will choose on its own.
 */
 #ifndef SQLITE_MAX_DEFAULT_PAGE_SIZE
-# define SQLITE_MAX_DEFAULT_PAGE_SIZE 8192
+# define SQLITE_MAX_DEFAULT_PAGE_SIZE 32768
 #endif
 #if SQLITE_MAX_DEFAULT_PAGE_SIZE>SQLITE_MAX_PAGE_SIZE
 # undef SQLITE_MAX_DEFAULT_PAGE_SIZE
